src/lib/stores/selectionStore.ts

```
@@ -0,0 +1,29 @@
+// src/stores/localStorageStore.js
+import type { TSelection } from '$lib/types';
+import { writable } from 'svelte/store';
+
+export function localStorageStore(key: string, initialValue: TSelection[]) {
+	let storedValue;
+
+	try {
+		const json = localStorage.getItem(key);
+		storedValue = json ? JSON.parse(json) : initialValue;
+	} catch (err) {
```

ESLint

[error] 'err' is defined but never used.

(@typescript-eslint/no-unused-vars)